layouts/default.vue

```diff
@@ -138,15 +119,18 @@
 import { mapState, mapGetters } from 'vuex'
 
 import ImportStudents from '@/components/ImportStudents.vue'
+import AsideBar from '@/components/AsideBar.vue'
 export default {
   middleware: ['auth'],
   components: {
-    ImportStudents
+    ImportStudents,
+    AsideBar
   },
   data() {
     return {
       active: this.isActive(),
       activateModal: false,
+      open: false,
```

Comment:
Só me confirma uma coisa. Se for mobile, tem que começar aberto? Caso sim, isso é alterado ou como funciona?

Reply:
Tu achas melhor começar logo aberto?

Reply:
Resolvido